src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -36,5 +36,36 @@ public IActionResult GetUser(Guid id)
             return Ok(user);
         }
 
+
+        
+        [HttpPost("")]
+        public  IActionResult PostUsers([FromBody] UserCreateDTO user)
+        {
```

Comment:
Why is this method called PostUsers? 

Can you suggest a better name for it, one that depicts the action being performed on a user?

Reply:
I called it postUsers because I read in a blog that it is useful to add the post prefix before the method name. 

I could also have named it InsertUsers.

Reply:
okay, fair enough. 

I'll suggest you rather use the actual "action word", which is "Create". 

So in this case, `CreateUser` sounds better.

You don't need to use Post in the name given that you already have the `HttpPost` attribute on the method.